sw/host/opentitanlib/src/test_utils/mod.rs

```diff
@@ -5,6 +5,9 @@
 pub mod bootstrap;
 pub mod e2e_command;
 pub mod epmp;
+// The "english breakfast" variant of the chip doesn't have the same
+// set of CSRs as the "earlgrey" chip.
+#[cfg(not(feature = "english_breakfast"))]
```

Comment:
Getting closer... Looks like this is needed here too:

https://github.com/lowRISC/opentitan/blob/24775ce73f0e9d70ed014c8560595e04cf79c053/sw/host/opentitanlib/src/dif/mod.rs#L6

Reply:
Done